INTRODUCTION.md

```diff
@@ -485,6 +486,45 @@ a topic's partition count is known, which would insert these messages
 after the partition-explicit messages regardless of produce order.
 
 
+#### Message timeout considerations
```

Comment:
`enable.gapless.guarantee` isn't mentioned at all in this section, i think it would be good if it was.